index.html

```diff
@@ -57,6 +57,257 @@ <h2>I’am a Software Developer</h2>
+                            </li>
+                            <li>
+                                <a href="#">bootstrap</a>
+                            </li>
+                            <li>
+                                <a href="#">Ruby</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="submit">See Project</button>
+
+                </div>
+            </div>
+            <div class="project-background">
+                <div class="project-foreground">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div class="project-keyword">
+                        <ul>
+
+                            <li>
+                                <a href="#">html</a>
+                            </li>
+                            <li>
+                                <a href="#">bootstrap</a>
+                            </li>
+                            <li>
+                                <a href="#">Ruby</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="submit">See Project</button>
+
+                </div>
+            </div>
+            <div class="project-background">
+                <div class="project-foreground">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div class="project-keyword">
+                        <ul>
+
+                            <li>
+                                <a href="#">html</a>
+                            </li>
+                            <li>
+                                <a href="#">bootstrap</a>
+                            </li>
+                            <li>
+                                <a href="#">Ruby</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="submit">See Project</button>
+
+                </div>
+            </div>
+
+        </div>
+
+    </section>
+    <section>
+        <div class="about">
+            <div class="about-header">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to
+                    contact me.</p>
+
+                <button type="submit">Get My Resume</button>
+                <hr>
+            </div>
```

Comment:
- [ ] Hey @JakariaJishan 👏, you did well on this project but you are missing the background image in your `about me section`. Please make sure to add it.